### PR TITLE
Add "Understanding unknownProperty in Ember" post

### PR DESCRIPTION
**Pull request by san650**

### CODE REVIEW

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+author:
+  name: Santiago Ferreira
+  email: santiago.ferreira@wyeworks.com
+  twitter_handle: san650
+  github_handle: san650
+  image: /images/team/santiago-ferreira.jpg
+  description: Developer working with Ruby, JavaScript and Ember.
+---
+
+The Ember object model provides support for dynamically resolving properties in
```
Comment (Axxiss) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: Not sure 100% `dynamically resolving properties` -> `resolving properties dynamically `

Comment (san650) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: @smartinez87 ^^?

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+  unknownProperty(key) {
+    var content = this.get('content');
+
+    return content.get(key);
+  }
+});
+
+var object = MyClass.create({
+  content: Ember.Object.cteate({})
```
Comment (eldano) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: create*

Comment (san650) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: :+1:

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+that name.
+
+This behavior of the Ember object model is very powerful since it allows you to
+implement _proxies_ very easily. This ability to add a level of indirection
+when reading or writing attributes can be used to integrate other browser
+libraries or APIs with Ember.
+
+## `localStorage` Example
+
+Let’s look at an example in which we will use this functionality to create a
```
Comment (eldano) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: Let's \*take a\* look

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown (hunk 1 above)
Comment (smartinez87) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: it's good as it is :+1:

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown (hunk 3 above)
Comment (juanazam) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: I think it's as good as the current wording :smile:

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown (hunk 1 above)
Comment (san650) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: :+1:

File: source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown (hunk 3 above)
Comment (san650) [inline source/_posts/2015-11-24-understanding-unknownproperty-in-ember.markdown]: :+1:

**Comment by san650** [comment]
Thank you @smartinez87 @eldano @Axxiss @juanazam

**Comment by spastorino** [comment]
@san650 seems good :+1:

**Pull request merged**